sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -705,7 +795,9 @@ object RocksDBConf {
 
     RocksDBConf(
       storeConf.minVersionsToRetain,
+      storeConf.minDeltasForSnapshot,
```

Comment:
Where is the conf entry defined for this ?

Reply:
We just reuse the conf of HDFS state store.

Reply:
Do we need to document this somewhere ? In general, you also need to document the options added in the SS programming guide.

Reply:
I added changelogCheckpointing.enabled to programming guide, minDeltasForSnapshot is an internal option which has never been exposed in the docs.

Reply:
cc - @HeartSaVioR - thoughts here ?

Reply:
Configs marked as "internal" are "intentionally" excluded from the documentation. If we think this option should be something which should be easily accessible for "all personas", we should just remove internal and that would work. Otherwise (e.g. we want to ensure the config to be adjusted from only experts/operators) it's still making sense to leave it as internal.

Reply:
Thanks, let's leave this config as internal for now.